io.github.celluloid_player.Celluloid.json

```diff
@@ -9,6 +9,7 @@
       "--socket=fallback-x11",
       "--socket=wayland",
       "--device=all",
+      "--filesystem=xdg-run/pipewire-0:ro",
```

Comment:
This seems questionable.

1. It hardcodes -0, which won't always be correct
2. I believe it will fail when it doesn't exist
3. My understanding is Pipewire audio *should* work without permissions.

I would like some clarification from upstream Pipewire or Flatpak.

Reply:
@TingPing, thanks for the response.

1. As I mentioned and linked in the top comment, I'm not setting a precedent here, this is done also in the Flatpak Chromium package and a couple of other packages. The only difference here is that I'm setting the bind mount as read-only to protect the socket from deletion by the sandbox.
3. Just works doesn't sound right. In the `flatpak-override.1` man page there nothing about PipeWire socket permission, and there's nothing about PipeWire audio in the xdg-desktop-portal docs. So this seems like an oversight.

Reply:
> As I mentioned and linked in the top comment, I'm not setting a precedent here, this is done also in the Flatpak Chromium package and a couple of other packages. 

Well the hardcoding is still wrong even if other packages is wrong. I will assume that means it probably works when it doesn't exist though.

> Just works doesn't sound right. In the flatpak-override.1 man page there nothing about PipeWire socket permission, and there's nothing about PipeWire audio in the xdg-desktop-portal docs. So this seems like an oversight.

To be clear I didn't mean it *does* just work, but I believe it *should* just work. As in that is the goal of the Pipewire project. Now that we are approaching Pipewire being used maybe this discussion needs to happen now in flatpak.

Reply:
@TingPing Thanks again for the reply. I guess it's better to push this back until we will get clarification from Flatpak's devs, I also don't want to include a brittle workaround. I'm closing this for the time being.